components/base/Typography.tsx

```diff
@@ -0,0 +1,211 @@
+  font-size: 20px;
+  font-weight: 300;
+  line-height: 26px;
+`;
+
+const StyledParagraph = styled.p`
+  ${ParagraphStyles}
+  font-size: 16px;
+  line-height: 28px;
+`;
+
+const StyledInlineCode = styled.code`
+  color: ${PARAGRAPH_TEXT};
+  font-family: ${MONACO};
+  font-size: 13px;
+  line-height: 20px;
+  border: 1px solid ${GREY_3};
+  border-radius: 5px;
+  background-color: ${GREY_1};
+  padding: 2px 5px;
+  &.dark {
+    background-color: ${PURPLE_DARK};
+    color: ${GREY_4};
+  }
+`;
+
+const StyledLink = styled.a`
+  color: ${ORANGE_PRIMARY};
+  &:hover {
+    text-decoration: underline;
+    cursor: pointer;
+  }
+`;
+
+// List styles
+const ListStyles = css`
+  ${ParagraphStyles}
+  font-size: 16px;
+  letter-spacing: 0;
+  line-height: 28px;
+`;
+
+// Prop types
+type Props = {
+  children: any;
+  isDark?: boolean;
+};
+
+export const PreHeader = ({ children, isDark }: Props) => (
+  <StyledPreHeader className={classnames({ dark: isDark })}>
+    {children}
+  </StyledPreHeader>
+);
+
+export const H1 = ({ children, isDark }: Props) => (
+  <StyledH1 className={classnames({ dark: isDark })}>{children}</StyledH1>
+);
+
+export const H2 = ({ children, isDark }: Props) => (
+  <StyledH2 className={classnames({ dark: isDark })}>{children}</StyledH2>
+);
+
+export const H3 = ({ children, isDark }: Props) => (
+  <StyledH3 className={classnames({ dark: isDark })}>{children}</StyledH3>
+);
+
+export const H4 = ({ children, isDark }: Props) => (
+  <StyledH4 className={classnames({ dark: isDark })}>{children}</StyledH4>
+);
+
+export const H5 = ({ children, isDark }: Props) => (
+  <StyledH5 className={classnames({ dark: isDark })}>{children}</StyledH5>
+);
+
+export const SubHeader = ({ children, isDark }: Props) => (
+  <StyledSubHeader className={classnames({ dark: isDark })}>
+    {children}
+  </StyledSubHeader>
+);
+
+export const Paragraph = ({ children, isDark }: Props) => (
+  <StyledParagraph className={classnames({ dark: isDark })}>
+    {children}
+  </StyledParagraph>
+);
+
+export const UnorderedList = ({ children, isDark }: Props) => (
+  <ul
+    css={css`
+      ${ListStyles}
+    `}
+    className={classnames({ dark: isDark })}
+  >
+    {children.map((item) => (
+      <li key={item.id}>{item.text}</li>
+    ))}
+  </ul>
+);
+
+export const OrderedList = ({ children, isDark }: Props) => (
+  <ol
+    css={css`
+      ${ListStyles}
+    `}
+    className={classnames({ dark: isDark })}
+  >
+    {children.map((item) => (
+      <li key={item.id}>{item.text}</li>
+    ))}
+  </ol>
+);
+
+export const BlockQuote = ({ children, isDark }: Props) => (
+  <blockquote
+    css={css`
+      ${ParagraphStyles}
+    `}
+    className={classnames({ dark: isDark })}
+  >
+    {children}
+  </blockquote>
+);
+
+export const InlineCode = ({ children, isDark }: Props) => (
+  <StyledInlineCode className={classnames({ dark: isDark })}>
+    {children}
+  </StyledInlineCode>
+);
+
+export const Link = ({ children }: Props) => (
```

Comment:
Is this supposed to handle `isDark`?

Reply:
Perhaps not, because the styles for both Dark and Light is the same for links, which is `color: ${ORANGE_PRIMARY}` Unless I'm missing something 🤔

Reply:
Ah ok cool, we can probably just do `export const Link = StyledLink` here then if you want.

Reply:
Actually, thinking of doing this: 
```suggestion
export const Link = ({ text, href }: LinkProps) => (
  <StyledLink href={href}>{text}</StyledLink>
);```

Reply:
What do you think?

Reply:
Hmm, I'd probably lean towards keeping the props the same as the built-in `<a>` since all we're really adding are styles. Why change things up/write more code if we don't have to?

This works too tho, we can always change it if we need to!